.github/workflows/client_update.yml

```diff
@@ -0,0 +1,22 @@
+name: Trigger Client Update
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - 'server/api/tumdev/**'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Trigger Client Repository Dispatch
+        uses: peter-evans/repository-dispatch@v2
```

Comment:
From a conceptual perspective:
Is there a reason why adding this extra step is better?

I think leaving the complexity of adding the PR in this repo might be a good approach, as this way the boundarys are clearer.

=> i.e. something like https://github.com/orgs/community/discussions/49745

Reply:
What exactly do you mean?